examples/purchase-tester/src/main/res/layout/package_card.xml

```diff
@@ -144,11 +144,11 @@
                     app:layout_constraintTop_toTopOf="parent"
                     app:layout_constraintBottom_toTopOf="@+id/package_buy_button"/>
 
+            <!-- TODO BC5 enable based on whether current base plan is active -->
             <com.google.android.material.button.MaterialButton
                     android:id="@+id/package_buy_button"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:enabled="@{!isActive}"
```

Comment:
Should we do the same for the purchase product button?

Reply:
oh oops, of course!